apps/web/pages/auth/login.tsx

```diff
@@ -50,7 +52,23 @@ export default function Login({
 }: inferSSRProps<typeof _getServerSideProps> & WithNonceProps) {
   const { t } = useLocale();
   const router = useRouter();
-  const methods = useForm<LoginValues>();
+
+  const loginFormSchema = z.object({
+    email: z.string().email(),
+    password: z.string().refine((value) => totpEmail || value.length > 0, {
+      message: "Password is required.",
```

Comment:
use i18n here like t("password_is_required"). you can check the codebase for some examples

Reply:
<img width="993" alt="Screenshot 2023-05-22 at 6 35 28 PM" src="https://github.com/calcom/cal.com/assets/53316345/209afc08-f5f8-4796-a9bb-35decbbde2c8">

You can use this in the file

Reply:
Thank you @Udit-takkar for the guidance, I have made some changes to support multi-language. I think we don't need to use this function because it feels like this function is mostly used for API response errors

Reply:
Would you like me to add all languages' translations for the message?